orion/evaluation/utils.py

```diff
@@ -5,12 +5,16 @@
 
 
 def from_pandas_contextual(df):
-    """ convert contextual `pd.DataFrame` to list of tuples.
+    """ Convert contextual ``pandas.DataFrame`` to list of tuples.
 
     Args:
-        * df (pd.DataFrame): contains start and end.
-    """
+        df (DataFrame): anomalies, passed as ``pandas.DataFrame``
```

Comment:
The description here should be put in a different line:

```
df (DataFrame):
    anomalies, passed as ``pandas.DataFrame``
    containing two columns: start and stop.
```

When there is only one line in the description it does not really matter, but when you break the line in two or more, if the first one is kept at the same level as the arg name it does something funky when it is rendered by Sphinx.

Reply:
Done, didn't know multilines were an issue